src/pages/K8s.vue

```diff
@@ -160,6 +167,7 @@ export default {
   mounted() {
     const that = this;
 
+    this.currentPort = ipcRenderer.sendSync('current-port');
```

Comment:
We shouldn't be using `sendSync` — that's a synchronous cross-process call.  Please make this async instead.

Reply:
done